src/Components/WebAssembly/WebAssembly.Authentication/src/Microsoft.AspNetCore.Components.WebAssembly.Authentication.csproj

```diff
@@ -63,29 +64,33 @@
 
   <Target Name="CompileInterop" Condition="'$(BuildNodeJS)' != 'false' AND '$(DesignTimeBuild)' != 'true'" Inputs="$(InteropCompilationCacheFile)" Outputs="@(YarnOutputs)">
     <Yarn Command="install --mutex network" WorkingDirectory="$(YarnWorkingDir)" IgnoreStandardErrorWarningFormat="$(IgnoreYarnWarnings)" />
-    <Yarn Command="run build:release" WorkingDirectory="$(YarnWorkingDir)" Condition="'$(Configuration)' == 'Release'" IgnoreStandardErrorWarningFormat="$(IgnoreYarnWarnings)" />
+    <Yarn Command="run build:production" WorkingDirectory="$(YarnWorkingDir)" Condition="'$(Configuration)' == 'Release'" IgnoreStandardErrorWarningFormat="$(IgnoreYarnWarnings)" />
     <Yarn Command="run build:debug" WorkingDirectory="$(YarnWorkingDir)" Condition="'$(Configuration)' == 'Debug'" IgnoreStandardErrorWarningFormat="$(IgnoreYarnWarnings)" />
 
+    <Message Importance="high" Text="@(_InteropBuildOutput->'Emitted %(FullPath)')" />
+
+  </Target>
+
+  <Target Name="IncludeCompileInteropOutput">
     <ItemGroup>
       <_InteropBuildOutput Include="$(YarnWorkingDir)dist\$(Configuration)\**" Exclude="$(YarnWorkingDir)dist\.gitignore" />
+    </ItemGroup>
 
-      <StaticWebAsset Include="@(_InteropBuildOutput->'%(FullPath)')">
-        <SourceType>Computed</SourceType>
-        <SourceId>$(PackageId)</SourceId>
-        <ContentRoot>$([MSBuild]::NormalizeDirectory('$(YarnWorkingDir)\dist\$(Configuration)'))</ContentRoot>
-        <BasePath>_content/$(PackageId)</BasePath>
-        <RelativePath>$([System.String]::Copy('%(RecursiveDir)%(FileName)%(Extension)').Replace('\','/'))</RelativePath>
-        <AssetKind>All</AssetKind>
-        <AssetMode>All</AssetMode>
-        <AssetRole>Primary</AssetRole>
-        <OriginalItemSpec>@(_InteropBuildOutput->'%(Identity)'))</OriginalItemSpec>
-      </StaticWebAsset>
+    <DefineStaticWebAssets Condition="@(_InteropBuildOutput) != ''"
+      SourceType="Computed"
+      SourceId="$(PackageId)"
+      ContentRoot="$(YarnWorkingDir)dist\$(Configuration)\"
+      BasePath="_content/$(PackageId)"
```

Comment:
If it's unintentional to have backslashes in some places and forward slashes in others, can they be made consistent?

Reply:
BasePath explictily uses forward slashes

Reply:
Oh yeah, it's part of a URL. Got it. Thanks.